File: app/src/main/java/fr/jbme/raiderioapp/data/model/character/MythicPlusBestRun.java
package fr.jbme.raiderioapp.data.model.character;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class MythicPlusBestRun {

    @SerializedName("dungeon")
    @Expose
    public String dungeon;
    @SerializedName("short_name")
    @Expose
    public String shortName;
    @SerializedName("mythic_level")
    @Expose
    public Integer mythicLevel;
    @SerializedName("completed_at")
    @Expose
    public String completedAt;
    @SerializedName("clear_time_ms")
    @Expose
    public Integer clearTimeMs;
    @SerializedName("num_keystone_upgrades")
    @Expose
    public Integer numKeystoneUpgrades;
    @SerializedName("map_challenge_mode_id")
    @Expose
    public Integer mapChallengeModeId;
    @SerializedName("score")
    @Expose
    public Float score;
    @SerializedName("affixes")
    @Expose
    public List<Affix> affixes = null;
    @SerializedName("url")
    @Expose
    public String url;

}

File: app/src/main/java/fr/jbme/raiderioapp/model/RIOCharacter/MythicPlusBestRun.kt
package fr.jbme.raiderioapp.model.RIOCharacter

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class MythicPlusBestRun(
    @SerializedName("characterDungeon")
    @Expose
    var dungeon: String? = null,

    @SerializedName("short_name")
    @Expose
    var shortName: String? = null,

    @SerializedName("mythic_level")
    @Expose
    var mythicLevel: Int? = null,

    @SerializedName("completed_at")
    @Expose
    var completedAt: String? = null,

    @SerializedName("clear_time_ms")
    @Expose
    var clearTimeMs: Int? = null,

    @SerializedName("num_keystone_upgrades")
    @Expose
    var numKeystoneUpgrades: Int? = null,

    @SerializedName("map_challenge_mode_id")
    @Expose
    var mapChallengeModeId: Int? = null,

    @SerializedName("score")
    @Expose
    var score: Float? = null,

    @SerializedName("affixes")
    @Expose
    var affixes: List<Affix>? = null,

    @SerializedName("url")
    @Expose
    var url: String? = null
)